### DISCUSSION TITLE
Replace Acorn with SWC parser for performance gains

### DISCUSSION THREAD

**Discussion by mohsen1** [webpack/webpack#15308] [Ideas]
I'm not sure how impactful this could be so I would like to know what @sokra and other team members think before starting to experiment with this idea. I'm assuming SWC parser and Acorn parsers output the same ESTree compatible ASTs, we can try out replacing the parser to measure performance benefits.

Thoughts?

**Comment by vankop** [comment]
there is an issue to track this https://github.com/webpack/webpack/issues/13425

**Reply by alexander-akait** [reply]
Yep, we can try to do it, @vankop I think we can introduce dangerous options (maybe under experiments) and allow to use it (you will need to install)
